integration/helm_test.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package integration
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/integration/skaffold"
+	"github.com/google/go-cmp/cmp"
+)
+
+const (
+	True = "true"
+)
+
+func TestHelmDeploy(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping integration test")
+	}
+	if os.Getenv("REMOTE_INTEGRATION") != True {
+		t.Skip("skipping remote only test")
+	}
+
+	ns, client, deleteNs := SetupNamespace(t)
+	defer deleteNs()
+
+	// To fix #1823, we make use of env variable templating for release name
+	env := []string{fmt.Sprintf("TEST_NS=%s", ns.Name)}
+	depName := fmt.Sprintf("skaffold-helm-%s", ns.Name)
+	helmDir := "examples/helm-deployment"
+	skaffold.Deploy().InDir(helmDir).InNs(ns.Name).WithEnv(env).RunOrFailOutput(t)
+
+	client.WaitForDeploymentsToStabilize(depName)
+
+	expectedLabels := map[string]string{
+		"app.kubernetes.io/managed-by": True,
```

Comment:
Could you rename `True` to something more expressive? This got me quite confused.

Reply:
sure, i can put a dummy version string.